package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "passport-auth0",
-  "version": "0.6.1",
+  "name": "@nudj/passport-auth0",
+  "version": "0.1.0",
```

Comment:
Discussion for us: I wonder whether we want to make this a non-semver style version. It would be really useful to know what the underlying version we rebased over is. Something like `0.6.1-0.1.0` or similar. This would allow us to track our own changes at the same time as the parent repo changes. If we go with this we'll need to use a slightly different regex in our CI settings in order to trigger the build - ask me for details if and when you come to setting this us.

Alternatively we could just rely on our git history but this requires a bit of sifting so harder to know just by looking at the version.

Any thoughts?